packages/pricing/src/migrations/Migration20230929122253.ts

```diff
@@ -3,19 +3,25 @@ import { Migration } from "@mikro-orm/migrations"
 export class Migration20230929122253 extends Migration {
   async up(): Promise<void> {
     this.addSql(
-      'create table if not exists "money_amount" ("id" text not null, "currency_code" text null, "amount" numeric null, "min_quantity" numeric null, "max_quantity" numeric null, constraint "money_amount_pkey" primary key ("id"));'
+      'create table if not exists "money_amount" ("id" text not null, "currency_code" text not null, "amount" numeric not null, "min_quantity" numeric null, "max_quantity" numeric null, constraint "money_amount_pkey" primary key ("id"));'
```

Comment:
q: Is there a plan to update the migration for backward compatibility? like we do if table exists then we update the table and otherwise we do the rest

Reply:
Yes, there is already a migration script that takes care of this. The changes made here is more aligned with the core table.